.gitignore

```diff
@@ -1,14 +1,12 @@
-**/*.iml
 *.log
-**/*.iml
+*.iml
```

Comment:
revert this change

Reply:
*.iml ignores the top level .iml file (which is only one we have in our structure). This change is from the commit that Matt Doherty had. So,  I will keep it to keep the .gitignore file consistent between branches.